src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -187,6 +163,7 @@ export default class HaAutomationActionRow extends LitElement {
           ${this.index !== this.totalActions - 1
             ? html`
                 <ha-icon-button
+                  slot="header"
```

Comment:
Same here, and obviously the same for the triggers and conditions.

Reply:
Fixed